signup.php

```diff
@@ -4,21 +4,46 @@
 
 include('config/init.php');
 echoHeader('Sign up');
+$errors=array();
+global $errors;
```

Comment:
I don't think you need line 8. It's not really hurting anything, but I think since you're not inside a function here, you're already in the global scope so you don't need to declare anything as a global variable.